content/quiz_events.md

```diff
@@ -4,21 +4,134 @@
+
 ### Correct
+The quizzer's response is correct when:
+- It fulfills all requirements specific to the Question Type.
+- The quizzer stays in Context.
+- It contains the information requested.
+    - If the quizmaster did not complete the reading of the question, the quizzer's answer must include the information in the remainder of the question as well as the answer.
+    - It is not required to be in question and answer form, except in the case of a reference question.
+    - The quizzer’s answer must include all unique words included in both the question and the answer.
+
+Mispronounced names are still recognizable as the answer.
+
+Only the first response of the quizzer will be considered, however, if the response by the quizzer is in context and is not incorrect, the quizzer will be allowed their full 30 seconds to give the additional information necessary to satisfy the question and the answer.  Example: INT Who has been making it grow? A. God.  If the quizzer said “Apollos” they have given incorrect information (even though the name “Apollos” is in context) and would be called incorrect.  If they had said “I planted the seed”, “Apollos watered it”, they have not given any incorrect information, are in context and will be given the 30 seconds to supply the correct answer required. 
 
 ### Incorrect
+The quizzer's response is incorrect when:
+- Incorrect information has been given.
+    - e.g If the wrong person (i.e. Father, Son, Holy Spirit) in the deity (God) is given.
+- The information given within the time limit is incomplete.
+- It does not fulfill all requirements specific to the Question Type.
+     - e.g. Any word is omitted or added in a Finish-The-Verse, Finish-This, or Quote Question or the wrong verse is begun.
 
 ### Challenges
+Challenges exist so quizzers can contest rulings made by quiz officials.
+
+The team captain or co-captain may challenge if:
+- There is reason to believe the quiz officials have applied the rule book incorrectly
+- There has been a mispronunciation by the quizmaster
+- There has been an invalid question
+- The next question has not been called
+    - A challenge must be made at the time of the quizmaster's decision and before the next question is called. If a time-out is called immediately after a ruling, and a quizzer stands to challenge, the challenge may be done prior to the time-out.
+- No conferring has occurred between the quizzer challenging and anyone else. Conferring with anyone else prior to a challenge will result in:
+    - The challenge being disallowed
+    - A foul being awarded to the quizzer.
+
+Once a team has requested an explanation of a ruling, and that explanation has been given, all teams lose the right to challenge.
+- If an explanation of a ruling is requested, the quizmaster should pause before giving the explanation, to allow all teams the opportunity to challenge.
```

Comment:
I added this....not sure where this best exists or how it should be worded.

Reply:
This is the best place for it.

Reply:
This does exist under QM right now "If an explanation of a ruling is requested by a captain, the quizmaster will ask if any team wants to challenge before the explanation is given."     BTW, does this mean only a captain can request an explanation of a ruling?

Reply:
I prefer the "ask if there are any challenges" over "pause before" language. The pause seems awkward. And I think in practice anyone can ask for a clarification.

Reply:
I don't know of QMs who refuse requests for clarification from non-captains. But the rulebook definitely limits this ability to captains.

Reply:
The "pause" bit was an oversight by me. I didn't realize elsewhere it says the QM "will ask" if anyone wants to challenge.

Reply:
Hm. OK. So do we want to consider this a bug in the rulebook or a bug in implementation? I'm inclined to consider it a bug in the rulebook, but I could be easily swayed.

Reply:
I don't think a bug exists. My attempt to port over section (x) and make it more clear led me to add the "pause" language. But later I found content in section (y) that overrode the lack of clarity in section (x), rendering my additions null and void.

Reply:
No, I mean a bug in how QMs don't limit requests for clarity to captains whereas the rulebook says it should be limited to captains. Right?

Reply:
I can't say whether it's a rule book oversight (rule book bug), or a QM implementation problem (QM bug).

I know what outcome I prefer (anyone can ask for a clarification). 

But I don't know what's closest to func equiv.

Reply:
I say let's assume it's a rulebook bug and go with what QMs do, which I think is both right and easiest.